packages/wdio-utils/package.json

```diff
@@ -33,6 +33,7 @@
     "@puppeteer/browsers": "^1.6.0",
     "@wdio/logger": "8.16.17",
     "@wdio/types": "8.17.0",
+    "@wdio/config": "8.18.0",
```

Comment:
This would introduce a circular dependency since `@wdio/config` also depends on `@wdio/utils`. Let's move `isAppiumCapability` into `@wdio/utils` and reference it in `@wdio/config`